addons/mrp/i18n/mrp.pot

```diff
@@ -3817,6 +3823,14 @@ msgid ""
 "date, please correct this to save the work order."
 msgstr ""
 
+#. module: mrp
+#: code:addons/mrp/models/product.py:0
+#, python-format
+msgid ""
+"The product type of %s must be 'Consumable' because it has at least one kit-"
```

Comment:
Why 2 lignes there and not in the original commit ?

Reply:
The forward port failed since there were some conflicts, including in the translations file (see c6d05deaf8f3c9054e5512d20634528e12bfda27). The current `mrp.pot` directly comes from a new translation export.